pkg/amber/provenance.go

```diff
@@ -100,6 +100,25 @@ func (p *ValidatedProvenance) GetBuildCmd() ([]string, error) {
 	return buildConfig.Command, nil
 }
 
+// GetConfigSourceURI returns the URI of the configuration.
+func (p *ValidatedProvenance) GetConfigSourceURI() string {
+	return p.provenance.Predicate.(slsa.ProvenancePredicate).Invocation.ConfigSource.URI
+}
+
+// GetMaterialsGitURI returns references to a Git repo.
+func (p *ValidatedProvenance) GetMaterialsGitURI() []string {
+	materials := p.provenance.Predicate.([]slsa.ProvenanceMaterial)
```

Comment:
Does this work? I think it should be changed to the following:
```suggestion
	materials := p.provenance.Predicate.Materials
```

Or better, change the signature to take a `slsa.ProvenancePredicate` as input. Similar to https://github.com/project-oak/transparent-release/blob/e6fbeb27dede4e91fa8553e38439f005c1639ccb/pkg/amber/provenance.go#L112.

Reply:
Done.